check/format-incremental

```diff
@@ -130,7 +130,7 @@ echo "Running the black formatter... (version: $BLACKVERSION)"
 
 args=("${arg_color[@]}")
 if (( only_print == 1 )); then
-    args+=("--check" "--diff")
+    args+=("--check" "--diff" "--ipynb")
```

Comment:
The `--ipynb` option makes `black` execution fail as it considers all files to be jupyter notebooks.  Please remove.

The `format_files` array needs to include the `.ipynb` files to get them checked and formatted.  Please make sure these do not leak to `isort_files`, because `isort` does not seem to work with notebooks.  For now, let isort look only at the python files.